hoot-core/src/main/cpp/hoot/core/ops/GeometryModifierOp.cpp

```diff
@@ -0,0 +1,168 @@
+/*
+ * This file is part of Hootenanny.
+ *
+ * Hootenanny is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * --------------------------------------------------------------------
+ *
+ * The following copyright notices are generated automatically. If you
+ * have a new notice to add, please use the format:
+ * " * @copyright Copyright ..."
+ * This will properly maintain the copyright information. DigitalGlobe
+ * copyrights will be updated automatically.
+ *
+ * @copyright Copyright (C) 2015, 2016, 2017, 2018, 2019 DigitalGlobe (http://www.digitalglobe.com/)
+ */
+
+#include "GeometryModifierOp.h"
+#include "hoot-core/src/main/cpp/hoot/core/visitors/geometrymodifiers/GeometryModifierAction.h"
+
+// Hoot
+#include <hoot/core/util/ConfigOptions.h>
+#include <hoot/core/util/Factory.h>
+#include <hoot/core/util/MapProjector.h>
+
+namespace bpt = boost::property_tree;
+using namespace boost;
```

Comment:
We've seen issues in parts of the code where we use the namespace `boost` and use `shared_ptr` without the boost prefix and have run into issues of it using the wrong shared pointer.  Especially if you include `using namespace std;`

Reply:
I had to make a choice whether I put `boost` or `std` and I opted for using `boost` since it is used more often, typing out `std::` where needed, to minimize clutter and avoid conflicts. 
Would you prefer me to not use `using` at all or prefer `std` over `boost` and type out `boost::`?

Reply:
In the codebase there are 22 instances of `using namespace boost` and 436 `using namespace std`.  In the first instance of using `boost` it didn't even seem to be using any boost in the file, must have been a leftover.  In the second the pointers were still referenced with `boost::shared_ptr` because of the collision between that and `std::shared_ptr` from C++11.  I wonder if we ought to begin replacing Boost shared pointers with standard library shared pointers.

Reply:
And by begin, I mean do it all at once because the two don't play nice together.

Reply:
I always meant to use the boost version. Seeing that that all the predefined Ptr types are the boost version I figured the std version is not intended anywhere.
I removed the using for now and made it explicit in this module.

I find the mix of std, boost and Qt in this code base extremely unfortunate. I think we ought to look at what each library is really needed for, keep only the most important one (probably either boost or Qt) as the choice for all general use (containers, shared pointers, multithreading, strings, etc.) and allow any other library only in localized places where their special functionality or implementation is absolutely needed. No mix of interfaces where on class needs std strings, the other QString, etc.

Reply:
This is one of the topics to discuss while we are in DC. :-)